hack/verify-generated-files-remake.sh

```diff
@@ -291,11 +291,14 @@ fi
 
 assert_clean
 
-touch "staging/src/k8s.io/api/core/v1/types.go"
+touch "staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go"
```

Comment:
should we touch both files?

Reply:
Actually I want both tests: apimachinery leads to all specs regenerated, core/v1 only to an updated main spec.

Reply:
done